CHANGES.rst

```diff
@@ -4,12 +4,22 @@ Change Log
 git master
 ----------
 
+New features
+''''''''''''
+
+* Added support to auto-generate Binder links for examples via ``binder``
+  config. `#244 <https://github.com/sphinx-gallery/sphinx-gallery/pull/244>`_
+
 Bug Fixes
 '''''''''
 
 * Fix __future__ imports across cells. See `#308
   <https://github.com/sphinx-gallery/sphinx-gallery/pull/308>`_ for more details.
 
+Developer changes
+'''''''''''''''''
+
+* Dropped support for Sphinx <= 1.4.
```

Comment:
can someone update me on this. Will we still test on 1.4.9?

Reply:
No. The `AssertionError len(context) = 1` thing has been fixed in sphinx 1.5. Not sure why we haven't seen it before in sphinx-gallery but this is something that we have seen in other projects (I think scikit-learn).